hw/ip/aes/data/aes.hjson

```diff
@@ -273,8 +277,10 @@
         '''
       }
     ]
-    tags: [// Updated by hw. Exclude from write-checks.
-           "excl:CsrNonInitTests:CsrExclWriteCheck"]
+    tags: [// Updated by the HW.
+           // Updates based on writes to other regs (also during reset test).
+           // Exclude from init and write-read checks.
+           "excl:CsrAllTests:CsrExclCheck"]
```

Comment:
just curious why reset can't change this reg back to default value. Is it same for all 4 fields in `Status`?

Reply:
A reset correctly sets the register to the default value. But writes to other registers can trigger various operations inside the module. In this case, `Status` changes and is no longer the default (even though `Status` has not been accessed before during the test). This causes those tests under some circumstances to fail.

Reply:
I also found below error in nightly regression.
> UVM_ERROR @   7193075 ps: (csr_utils_pkg.sv:381) [csr_utils::csr_rd_check] Check failed obs == exp (8 [0x8] vs 9 [0x9]) Regname: ral.status 
> UVM_INFO @   7193075 ps: (uvm_report_catcher.svh:705) [UVM/REPORT/CATCHER] 

After a closer look, I think we need to change seq. The "CsrInitTests" only checks default value after reset, it shouldn't fail, since `Status` can be reset to default value. We should still use `CsrNonInitTests` for `Status` and also change tag for TRIGGER to "excl:CsrNonInitTests:CsrExclCheck".

In [aes_base_vseq](https://github.com/lowRISC/opentitan/blob/master/hw/ip/aes/dv/env/seq_lib/aes_base_vseq.sv), we add `aes_init` in `dut_init` which writes 2 registers. `dut_init` is also called in CSR seq after reset. These writes change the value of regs (CSR seq isn't aware of the change), so we see mismatch in CSR seq.
```
  virtual task dut_init(string reset_kind = "HARD");
    super.dut_init();
    if (do_aes_init) aes_init();
    aes_item_init();
  endtask
  virtual task aes_init();
   ...
    csr_wr(.csr(ral.ctrl), .value(aes_ctrl));
    csr_wr(.csr(ral.trigger), .value(aes_trigger));
  endtask
```

@rasmus-madsen can we either move `aes_init` to seq body or set  `do_aes_init = 0` in [aes_common_vseq](https://github.com/lowRISC/opentitan/blob/master/hw/ip/aes/dv/env/seq_lib/aes_common_vseq.sv) for CSR seq.

Reply:
Ah, I see. Thanks @weicaiyang for explaining the situation. I will discuss with @rasmus-madsen and update this PR accordingly.